scripts/changelog.sh

```diff
@@ -150,33 +146,22 @@ function nextminor {
     NEXT_VERSION=$(npx -y changie@$CHANGIE_VERSION next minor)
     # Remove all existing per-release changelogs
     rm ./.changes/*.*.*.md
-    # Remove all unreleased changes
-    rm ./.changes/unreleased/*.yaml
-    # Remove all backported changes
-    rm ./.changes/backported/*.yaml
+    
     # Create a new empty version file for the next minor version
     touch ./.changes/$NEXT_VERSION.md
     
-    generate "dev"
-}
-
-# This function is expected to be run on the branch of the last minor release. It will make sure
-# that backports work properly
-function firstbeta {
-    # For the maintenance branch we don't want to base our changelog on the unreleased but the backported folder instead
-    awk '{sub(/unreleasedDir: unreleased/, "unreleasedDir: backported")}1' ./.changie.yaml > temp && mv temp ./.changie.yaml
-    
-    # If we have backported changes, we need to remove them now since they were backported into the
-    # last version
-    rm -f ./.changes/backported/*.yaml
+    LATEST_MAJOR_MINOR=$(echo $LATEST_VERSION | awk -F. '{print $1"."$2}')
+    NEXT_MAJOR_MINOR=$(echo $NEXT_VERSION | awk -F. '{print $1"."$2}')
     
-    # If we have unreleased changes, they will be released in the next patch, therefore they need
-    # to go into the backported folder
-    if [ "$(ls -A ./.changes/unreleased/)" ]; then
-        mv ./.changes/unreleased/* ./.changes/backported/
-    fi
+    # Create a new changes directory for the next minor version
+    mkdir ./.changes/v$NEXT_MAJOR_MINOR
+    touch ./.changes/v$NEXT_MAJOR_MINOR/.gitkeep
 
+    # Set changies changes dir to the new version
+    awk "{sub(/unreleasedDir: v$LATEST_MAJOR_MINOR/, \"unreleasedDir: v$NEXT_MAJOR_MINOR\")}1" ./.changie.yaml > temp && mv temp ./.changie.yaml
     generate "dev"
+    
+    echo "\n\n Please clean up the .changes folders that are no longer needed, but keep the .gitkeep file for the last two versions to enable backporting."
```

Comment:
nit: I see the `\n\n` in the text printed to the terminal, instead of whitespace